src/Tile/tilesObject.js

```diff
@@ -0,0 +1,103 @@
+import { env } from 'decentraland-commons'
```

Comment:
I don't like the name of this file/object but couldn't came up with something better. It's a tiles object indexed by id, with caching

Reply:
If the implementation is more clear, I would go with `cachedTiles` and `indexedTiles`

Why not keep it as `tiles.js`? For me is understandable that it manages tiles object and anything related as _cached_

Reply:
Hm...agreed think it's a good middle ground for now 💯